### PR TITLE
Docs: Style updates for Quarkus and Node.js quick starts

### PR DESCRIPTION
**Pull request by jbyrne-redhat**
**Notes for reviewers:**

- This PR applies style updates per our [MAS Documentation Style Guidelines](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit).
- A decision on how we refer to sections of the Red Hat Hybrid Cloud Console (such as the Service Accounts page) is pending further discussion. For now, I've made a proposal here of linking directly to it.
- Apart from that, everything else here is ready for review. 
- HTML previews are available for review [here](http://file.emea.redhat.com/~jbyrne/JK-9000/quarkus/README.html) and [here](http://file.emea.redhat.com/~jbyrne/JK-9000/nodejs/README.html).

### CODE REVIEW

File: docs/kafka/nodejs-kafka/README.adoc [hunk 1]
```diff
@@ -197,29 +194,31 @@ endif::[]
...
 
 .Prerequisites
-* You've created a Kafka instance in {product-kafka} and the instance is in the *Ready* state.
+* You've created a Kafka instance in {product-long-kafka} and the instance is in the *Ready* state.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to add a topic to.
-. Click *Create topic* and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
+. In the {product-kafka} web console, select *Kafka Instances* and then click the name of the Kafka instance that you want to add a topic to.
+. Select the *Topics* tab, click *Create topic*, and follow the guided steps to define the topic details, as shown in the figure.
```
Comment (AedinC) [inline docs/kafka/nodejs-kafka/README.adoc:201]: I notice that we've got some steps like this (throughout our documentation) where we role three steps into 1. I'd like to break this step up into two parts for increased readability:

- Select the **Topics** tab.
- Click **Create topic** and follow the guided steps to define the topic details as shown in the following figure.

File: docs/kafka/nodejs-kafka/README.adoc [hunk 2]
```diff
@@ -288,18 +284,16 @@ $ npm install
...
 . Run the producer component.
 +
-.Running the producer component
 [source]
 ----
 $ node producer.js
 ----
 +
-You should see output like that shown in the example.
+When the producer component runs, you see output like that shown in the example.
```
Comment (AedinC) [inline docs/kafka/nodejs-kafka/README.adoc:292]: Suggest add 'following' before example.

File: docs/kafka/nodejs-kafka/README.adoc [hunk 3]
```diff
@@ -315,12 +309,11 @@ As shown in the example, the producer component runs and generates messages that
 
 . Switch back to the first command-line window.
 +
-You should now see that the consumer component displays the same country names generated by the producer, and in the same order, as shown in the example.
+You now see that the consumer component displays the same country names generated by the producer, and in the same order, as shown in the example.
```
Comment (AedinC) [inline docs/kafka/nodejs-kafka/README.adoc:312]: Again, suggest adding 'following' before 'example'.

File: docs/kafka/quarkus-kafka/README.adoc [hunk 4]
```diff
@@ -228,9 +229,9 @@ The Quarkus example application in this quick start has three application-scoped
 $ cd ~/code-examples/quarkus-kafka-quickstart
 $ ./mvnw quarkus:dev
 ----
-. After the application is running, in a web browser, go to http://localhost:8080/prices.html[^] and verify that the `Last price` is updated.
+. When the application is running, in a web browser, go to http://localhost:8080/prices.html[^] and verify that the `Last price` value is updated.
```
Comment (AedinC) [inline docs/kafka/quarkus-kafka/README.adoc:232]: This sentence is a little awkward to read. Suggestion for increased readability:
When the application is running, perform the following actions:
-  Open a web browser and go to http://localhost:8080/prices.html. 
-  Verify that the `Last price` value is updated.

**Review by AedinC — APPROVED** [review]
Hi John. Great set of updates - just a couple of additional suggestions. Also, I can't seem to add a comment on any lines above 117, but I ran your previews through [Red Hat Vale](https://vale.vercel.app/), and it says that VSCode should be referred to by its full name- Visual Studio Code (line 96 in both files).

File: docs/kafka/nodejs-kafka/README.adoc [hunk 5]
```diff
@@ -142,15 +141,15 @@ endif::[]
...
 
-In this task, you'll create a new configuration file called `rhoas.env`. In this file, you'll set the required bootstrap server and client credentials as environment variables.
+In this task, you'll create a new configuration file called `rhoas.env`. In the file, you'll set the required bootstrap server and client credentials as environment variables.
 
 .Prerequisites
 ifndef::qs[]
 * You have the bootstrap server endpoint for your Kafka instance. To get the server endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots), and click *Connection*.
-* You have the generated credentials for your service account. To regenerate the credentials, use the *Service Accounts* page in the {product-kafka} web console to find your service account and update the credentials.
+* You have the generated credentials for your service account. To regenerate the credentials, use the https://console.redhat.com/application-services/service-accounts[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console to find your service account and update the credentials.
```
[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply docs/kafka/nodejs-kafka/README.adoc:152]: Yep. TBH, this sentence is a bit of a mouthful anyway. I think a nice cleanup would be:

```
You have the generated credentials for your service account. To reset the credentials, use the https://console.redhat.com/application-services/service-accounts[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
```

File: docs/kafka/nodejs-kafka/README.adoc (hunk 1 above)
Comment (jbyrne-redhat) [reply docs/kafka/nodejs-kafka/README.adoc:201]: Yep, I agree. I'll update just these two guides in this PR and make a note that we need to create an issue to update others.

File: docs/kafka/quarkus-kafka/README.adoc (hunk 4 above)
Comment (jbyrne-redhat) [reply docs/kafka/quarkus-kafka/README.adoc:232]: Yes, that's a nice update.

**Comment by jbyrne-redhat** [comment]
> 

Thanks for your review, @AedinC. Similar to https://github.com/redhat-developer/app-services-guides/pull/559#discussion_r933087515, I'll restrict updates of 'VSCode' to just the two guides covered by this PR. We can create a separate issue to track down and update other instances.

**Pull request merged**